src/main/java/dk/bec/polonez/reservationsystem/service/ReservationService.java

```diff
@@ -105,4 +124,23 @@ public boolean deleteReservation(Long id) {
         return true;
     }
 
+    @Scheduled(fixedRate = 1, timeUnit = TimeUnit.DAYS)
+    public void notifyAboutUpcomingEvents() {
+        long timestamp = System.currentTimeMillis() + 7 * 24 * 60 * 60 * 1000;
```

Comment:
**advice** A lot of magic numbers and maths here. What about using ```LocalDateTime.now()``` for fetching current system time?

Reply:
Got it, done